### PR TITLE
Adjust RuntimeTests retry count

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Build or CI related changes

## What is the new behavior?

Adjust tests retry count to be 3 for all tests, except RuntimeTests. RuntimeTests cannot be retried individually and do not benefit from larger retries. Automatic retry from the CI pipeline is better suited.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyPropertyGenerator.cs [hunk 1]
```diff
@@ -492,7 +492,7 @@ private static void GeneratePropertyStorage(IndentedStringBuilder builder, strin
 				}
 			}
 
-			private IDisposable GenerateNestingContainers(IndentedStringBuilder builder, INamedTypeSymbol typeSymbol)
+			private IDisposable GenerateNestingContainers(IndentedStringBuilder builder, INamedTypeSymbol? typeSymbol)
```
Comment (davidjohnoliver) [inline src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyPropertyGenerator.cs:495]: 👍

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ResourceOwner.cs [hunk 2]
```diff
@@ -3,6 +3,11 @@
 	/// <summary>
 	/// Resource owner variable definition
 	/// </summary>
+	/// <remarks>
+	/// The ResourceOwner scope is used to propagate the top-level owner of the resources
```
Comment (davidjohnoliver) [inline src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ResourceOwner.cs:7]: 👍 👍 👍

**Review by davidjohnoliver — APPROVED** [review]

File: src/SamplesApp/SamplesApp.UITests/RuntimeTests.cs [hunk 3]
```diff
@@ -19,7 +19,7 @@ public partial class RuntimeTests : SampleControlUITestBase
 		private readonly TimeSpan TestRunTimeout = TimeSpan.FromMinutes(2);
 
 		[Test]
-		[AutoRetry]
+		[AutoRetry(tryCount: 1)]
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/RuntimeTests.cs:22]: Wasn't aware there's a count on the attribute.  Nice!

**Review by carldebilly — APPROVED** [review]

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ResourceOwner.cs (hunk 2 above)
Comment (jeromelaban) [reply src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/ResourceOwner.cs:7]: 😁

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/_Tracing.cs [hunk 4]
```diff
@@ -10,7 +10,7 @@ namespace Microsoft.UI.Xaml.Controls
 {
 	internal static partial class _Tracing
 	{
-		[Conditional("TRACE")]
+		[Conditional("REPEATER_TRACE_ENABLED")]
```
Comment (dr1rrb) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/_Tracing.cs:13]: Personally I prefix all tracing compiler variable by "TRACE_".For instance here I would use "TRACE_ITEMSREAPETER" (I already have a couple of variable like that in Uno)

**Pull request merged**